app/Http/Controllers/SettingsController.php

```diff
@@ -194,8 +194,10 @@ public function exportToSql()
 
         $adapter = disk_adapter(ExportAccountAsSQL::STORAGE);
 
+        $exportdate = date('Y-m-d');
```

Comment:
Like phpstan report mentioned, you need to add `use function Safe\date;` at the beginning of the file

Reply:
I've given it a go in the newest commit.  I don't actually know any PHP, so this was done by combining Googled results with trial and error.  It tested successfully on my Monica instance without `Safe\date;`, but I've added it.  Hope this works!

Reply:
'Safe' is a package we use - see https://github.com/thecodingmachine/safe